.travis.yml

```diff
@@ -16,12 +16,22 @@ matrix:
       env:
         # Give a useful display name in the job list.
         - SHARD=rustfmt
-        - PATH=$HOME/.cargo/bin/$PATH
+        - PATH=$HOME/.cargo/bin/:$PATH
       before_script:
         - rustup component add --toolchain stable rustfmt-preview
       script:
         - echo "Checking Gotham codebase with rustfmt release `cargo fmt --version`."
         - cargo fmt --all -- --write-mode=diff
+    # Run Clippy in its own shard.
+    - rust: nightly-2018-06-29
+      env:
+        - SHARD=clippy
+        - PATH=$HOME/.cargo/bin/:$PATH
+      before_script:
+        - cargo install clippy --version 0.0.211
```

Comment:
I think it's ok to not pin a version here - we can just use latest, seeing as it's CI.

Reply:
There is a problem with current Clippy: latest Clippy not always compiles with latest nightly Rust, because internal APIs used by Clippy are constantly changing.

So I think it will be better to pin specific version of nightly Rust and specific version of Clippy until Clippy will be added to the rustup as a separate component (like rustfmt).

Reply:
@im-0 that seems reasonable, well thought out!